archeryutils/classifications/agb_indoor_classifications.py

```diff
@@ -292,7 +292,7 @@ def agb_indoor_classification_scores(
     # Handle possibility of gaps in the tables or max scores by checking 1 HC point
     # above current (floored to handle 0.5) and amending accordingly
     for i, (score, handicap) in enumerate(
-        zip(int_class_scores, group_data["class_HC"]),
+        zip(int_class_scores, group_data["class_HC"], strict=False),
```

Comment:
`strict=False` is the default and the previous behaviour for zip so technically not needed, but if you want to have it here to be extra explicit then no problem.

Reply:
This should be set to True, though, so will amend now - see https://github.com/jatkinson1000/archeryutils/pull/82/commits/f3c830afee6b30816c0eafb960ebfdff5fa88865